sentry-android-core/src/main/java/io/sentry/android/core/ScreenshotEventProcessor.java

```diff
@@ -52,6 +59,16 @@ public ScreenshotEventProcessor(
       return event;
     }
 
+    final boolean shouldDebounce = debouncer.checkForDebounce();
```

Comment:
I think everything is fine here
If you could just add a comment to explain the order of the calls it would be even better 😄